e2e/stack.go

```diff
@@ -211,6 +213,12 @@ func (s *Stack) runMonomer(ctx context.Context, env *environment.Env, genesisTim
 	env.DeferErr("close tx db", txdb.Close)
 	mempooldb := dbm.NewMemDB()
 	env.DeferErr("close mempool db", mempooldb.Close)
+	rawstatedb := rawdb.NewMemoryDatabase()
```

Comment:
one-time var

Reply:
we need this as a separate var so we can [close it](https://github.com/polymerdao/monomer/pull/72/files/e93ca84aec578d548311912b0870aa318b2074c6#diff-a60e40fd7c6abc129086b0e1217ee5295a4e4cbff6510b6f5fd253faaad6b553R221) after the test runs

Reply:
Got it, thanks. Can we move the defer right after this assignment? That way if we hit the error case, we still close the db.